src/components/SvgIcon.vue

```diff
@@ -27,12 +27,21 @@ export default class SvgIcon extends Vue {
   }
 
   get iconPath() {
-    let icon = require(`@/assets/images/svg/${this.name}.svg`);
-    if (Object.prototype.hasOwnProperty.call(icon, "default")) {
-      icon = icon.default;
-    }
+    if (this.name) {
```

Comment:
hmm, I don't remember exactly, but `name` turned out to be `null` in some cases. It could have been the result of using the same local storage between main and devnet during development.